spec/controllers/tournaments_controller_spec.rb

```diff
@@ -127,40 +140,58 @@
       context 'with existing teams' do
         it 'creates a new Tournament' do
           expect do
-            post :create, params: create_data
+            post :create, params: create_playoff_tournament_data
           end.to change(Tournament, :count).by(1)
         end
 
         it 'associates the new tournament with the authenticated user' do
           expect do
-            post :create, params: create_data
+            post :create, params: create_playoff_tournament_data
           end.to change(@user.tournaments, :count).by(1)
         end
 
         it 'associates the given teams with the created tournament' do
-          post :create, params: create_data
+          post :create, params: create_playoff_tournament_data
           body = deserialize_response response
           tournament = Tournament.find(body[:id])
           expect(tournament.teams).to match_array(@teams)
         end
 
         it 'generates a playoff stage' do
-          post :create, params: create_data
+          post :create, params: create_playoff_tournament_data
           body = deserialize_response response
           tournament = Tournament.find(body[:id])
           expect(tournament.stages.first).to be_a(Stage)
         end
 
         it 'generates a playoff stage with all given teams' do
-          post :create, params: create_data
+          post :create, params: create_playoff_tournament_data
           body = deserialize_response response
           tournament = Tournament.find(body[:id])
           included_teams = tournament.stages.first.matches.map { |m| m.match_scores.map(&:team) }.flatten.uniq
           expect(included_teams).to match_array(@teams)
         end
 
+        context 'generates a group stage tournament' do
+          before(:each) do
+            post :create, params: create_group_tournament_data
+            body = deserialize_response response
+            @group_stage_tournament = Tournament.find(body[:id])
```

Comment:
is there a reason this var needs a class-scope?

Reply:
It is instance scope, but I did it because in all the other tests the variables in other befores are all with `@` I can change it to `group_stage_tournament` as well. Why are all the others that way then?

Reply:
Because in other specs there's no `before(:each)` but a before for the complete spec and the vars are used in different contexts

Reply:
Well, I just realized I cannot change it to `group_stage_tournament`, as within the test it will say "Cannot find ".."" So what should I change it to?

Reply:
I think removing the `:each` and leaving the var in the current scope is the best option

Reply:
so like [this](https://github.com/turniere/turniere-backend/pull/35/commits/d89785f65a4223eb596b5f4f43773855ce12b332)?